docs/whats-new/preview-6.md

```diff
@@ -0,0 +1,189 @@
+---
+title: .NET Aspire preview 6
+description: .NET Aspire preview 6 is now available and includes many improvements and new capabilities.
+ms.date: 04/22/2024
+---
+
+# .NET Aspire preview 6
+
+.NET Aspire preview 6 introduces changes that improve the security and reliability of .NET Aspire. The following article provides an overview of the major changes in .NET Aspire preview 6: **8.0.0-preview.6.24214.1**.
+
+To update to the latest version, run the following commands:
+
+```dotnetcli
+dotnet workload update
+```
+
+```dotnetcli
+dotnet workload install aspire
+```
+
+Run `dotnet workload list` to see the installed workloads.
+
+```dotnetcli
+Installed Workload Id    Manifest Version                   Installation Source
+--------------------------------------------------------------------------------------------
+aspire                   8.0.0-preview.6.24214.1/8.0.100    SDK 8.0.200, VS 17.10.34804.81
+```
```

Comment:
Should this be swapped around and have some clearer intros?
- Check if it's installed and you're at the latest
- To update do ...
- To install do ...

Reply:
No, it works a bit differently. In that the workload update verifies latest manifest, and the workload install, installs said based on the manifest.